cores/arduino/stm32/PeripheralPins.h

```diff
@@ -71,5 +71,8 @@ extern const PinMap PinMap_Ethernet[];
 //*** QUADSPI ***
 extern const PinMap PinMap_QUADSPI[];
 
+//*** DMA ***
+extern const DmaMap DmaMap_DMA[];
+
```

Comment:
as suggested by @fpistm the DmAMap should be declared and managed in dma.c, not in PeripheralPins as there is no pin in this map.